app/controllers/bank_accounts_controller.rb

```diff
@@ -3,11 +3,17 @@ class BankAccountsController < ApplicationController
   before_action :set_bank_account, only: [:edit, :update, :destroy, :hide]
   before_action :require_organization, only: [:edit, :update, :new, :create,
     :destroy, :hide]
+  before_action :redirect_for_not_ready_organization, only: [:edit, :update, :new, :create,
```

Comment:
@dissident may be it will be better to add this filter by default in application_controller and just skip where you don't need it.